packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/Sell/AddBankDetails/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import AddBankDetails from './template.js'
+
+class AddBankDetailsContainer extends React.PureComponent {
+  render () {
+    return <AddBankDetails {...this.props} />
+  }
+}
+
+const mapStateToProps = (state) => ({
+})
```

Comment:
Does this need to be a connected component if there's nothing in `mapState` or `mapDispatch`?

Reply:
No, indeed I just created this file with all the basics for a connected component before knowing whether or not it would need `mapStateToProps` and `mapDispatchToProps` but it turns out they don't